Documentation/contributing/development/dev_setup.rst

```diff
@@ -554,6 +554,17 @@ And then:
 
 Now new containers will have an IPv6 address assigned to them.
 
+Updating cmdref
+~~~~~~~~~~~~~~~
+
+If you change CLI arguments of any binaries in this repo, the CI will reject your PR if you don't
+also update the command reference docs. To do so, make sure to build binaries:
```

Comment:
```suggestion
also update the command reference docs. To do so, make sure to build the binaries and to run the relevant script:
```

Reply:
Nit on the nit: the second `to` isn't required here:
```suggestion
also update the command reference docs. To do so, make sure to build the binaries and run the relevant script:
```

Reply:
:detective: